src/rsz/src/Resizer.cc

```diff
@@ -843,7 +843,8 @@ Resizer::repairNet(Net *net,
         const RiseFall *tr1;
         sta_->checkCapacitance(drvr_pin, nullptr, max_,
                                corner1, tr1, cap1, max_cap1, cap_slack1);
-        if (corner1) {
+        if (max_cap1 > 0.0 // medhi proofing
```

Comment:
This comment might not be clear to future or current authors. Perhaps adding a comment about what caused this issue to arise, and how these checks prevent that situation would be more helpful.